fleet-cmdr/src/handlers/cmd_orb_details.rs

```diff
@@ -0,0 +1,49 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+use orb_relay_client::{QoS, RecvMessage};
+use orb_relay_messages::{
+    orb_commands::v1::{OrbCommandError, OrbDetailsRequest, OrbDetailsResponse},
+    prost::Message,
+};
+use tracing::info;
+
+use super::OrbCommandHandler;
+
+#[derive(Debug)]
+pub struct OrbDetailsCommandHandler {}
+
+impl OrbDetailsCommandHandler {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl OrbCommandHandler for OrbDetailsCommandHandler {
+    #[tracing::instrument]
+    async fn handle(&self, command: &RecvMessage) -> Result<(), OrbCommandError> {
+        info!("Handling orb details command");
+        let _request = OrbDetailsRequest::decode(command.payload.as_slice()).unwrap();
+        // TODO(paulquinn00): Consult with @oldgalileo and @sfikastheo to determine where to get this info from.
+        let response = OrbDetailsResponse {
+            orb_id: "".to_string(),
+            orb_name: "".to_string(),
+            jabil_id: "".to_string(),
+            hardware_version: "".to_string(),
+            software_version: "".to_string(),
+            software_update_version: "".to_string(),
+            os_release_type: "".to_string(),
+            active_slot: "".to_string(),
```

Comment:
Environment variable filled in by `orb-slot-ctrl`, or through the `orb-slot-ctrl` library.